docs/async/index.md

```diff
@@ -0,0 +1,267 @@
+  "state": "STORE_RESIDENT",
+  "start_time_in_millis": 1599833907465,
+  "expiration_time_in_millis": 1600265907465,
+  "response": {
+    "took": 83,
+    "timed_out": false,
+    "_shards": {
+      "total": 20,
+      "successful": 20,
+      "skipped": 0,
+      "failed": 0
+    },
+    "hits": {
+      "total": {
+        "value": 1000,
+        "relation": "eq"
+      },
+      "max_score": 1,
+      "hits": [
+        {
+          "_index": "bank",
+          "_type": "_doc",
+          "_id": "1",
+          "_score": 1,
+          "_source": {
+            "email": "amberduke@abc.com",
+            "city": "Brogan",
+            "state": "IL"
+          }
+        },
+       {....}
+      ]
+    },
+    "aggregations": {
+      "city": {
+        "doc_count_error_upper_bound": 0,
+        "sum_other_doc_count": 997,
+        "buckets": [
+          {
+            "key": "belvoir",
+            "doc_count": 2
+          },
+          {
+            "key": "aberdeen",
+            "doc_count": 1
+          },
+          {
+            "key": "abiquiu",
+            "doc_count": 1
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+After the response is successfully persisted, you get back the `STORE_RESIDENT` state in the response.
+
+You can poll the ID with the `wait_for_completion_timeout` parameter to wait for the results received for the time that you specify.
+
+For asynchronous searches with `keep_on_completion` as `true` and a sufficiently long `keep_alive` time, you can keep polling the IDs until the search finishes. If you don’t want to periodically poll each ID, you can retain the results in your cluster with the `keep_alive` parameter and come back to it at a later time.
+
+## Delete searches and results
+
+You can use the DELETE API operation to delete any ongoing asynchronous search by its ID. If the search is still running, it’s canceled. If the search is complete, the saved search results are deleted.
+
+```json
+DELETE _opendistro/_asynchronous_search/<ID>?pretty
+```
+
+#### Sample response
+
+```json
+{
+  "acknowledged": "true"
+}
+```
+
+## Monitor stats
+
+You can use the stats API operation to monitor asynchronous searches that are running, completed, and/or persisted.
+
+```json
+GET _opendistro/_asynchronous_search/_stats
+```
+
+#### Sample response
+
+```json
+{
+  "_nodes": {
+    "total": 8,
+    "successful": 8,
+    "failed": 0
+  },
+  "cluster_name": "264071961897:asynchronous-search",
+  "nodes": {
+    "JKEFl6pdRC-xNkKQauy7Yg": {
+      "name": "40eae75ed3fa9633fec88fd60ec189fb",
+      "transport_address": "10.212.60.202:9300",
+      "host": "10.212.60.202",
+      "ip": "10.212.60.202:9300",
+      "roles": [
+        "data",
+        "ingest",
+        "remote_cluster_client"
+      ],
+      "attributes": {
+        "distributed_snapshot_deletion_enabled": "false",
+        "zone": "us-east-1c",
+        "cross_cluster_transport_address": "2600:1f18:10d4:8802:91e6:5857:5a69:9348",
+      },
```

Comment:
We have skipped this section to redact further details. We can skip this section